backend/arm64/emit.mlp

```diff
@@ -1046,9 +1045,17 @@ let emit_instr i =
         end
     | Lop(Const_int n) ->
         emit_intconst i.res.(0) n
-    | Lop (Const_float32 _) ->
-        (* CR mslater: (float32) arm64 *)
-        Misc.fatal_error "float32 is not supported on this architecture"
+    | Lop (Const_float32 f) ->
+        DSL.check_reg Float32 i.res.(0);
+        if f = 0l then
```

Comment:
(I am surprised this comparison does
not trigger an error - should the
polymorphic comparison be shadowed
in this file?)